src/signalcorr.jl

```diff
@@ -124,6 +128,8 @@ be subtracted from `x` before computing the ACF.
 If `x` is a vector, `r` must be a vector of the same length as `x`.
 If `x` is a matrix, `r` must be a matrix of size `(length(lags), size(x,2))`, and
 where each column in the result will correspond to a column in `x`.
+
+The output is normalized by the variance of `x`. See `autocov!` if you do not want this normalization.
```

Comment:
The second sentence still isn't consistent with the one above. Sorry, I'm not on my computer right now so I can't make edits directly. (Anyway, what takes time and can sound inefficient is to discuss the best formulation. Who makes the final edit after everybody agrees is secondary.)